packages/fluentui/react-builder/src/components/PropSelector.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { Dialog, Flex, RadioGroup, Divider, Text, Input } from '@fluentui/react-northstar';
+
+export const PropSelector: React.FunctionComponent<{ components: any; onConfirm: any; onCancel: () => void }> = ({
+  components,
+  onConfirm,
+  onCancel,
+}) => {
+  console.log(JSON.parse(JSON.stringify(components)));
```

Comment:
Why are you doing a deep copy for a console.log ? maybe I am missing some context here of what `component` might be.

Similarly in the line below, it doesn't look like you are doing anything with `components`

Reply:
Thank you, I forgot to remove it